server/pubEdgeProposal/api.js

```diff
@@ -28,6 +27,6 @@ app.get(
 			edge = await createPubEdgeProposalFromCrossrefDoi(object);
 		}
 
-		return edge ? res.status(200).json(edge) : res.status(404);
+		return res.status(200).json(edge);
```

Comment:
I forgot that when `apiFetch` gets a non-2XX status code, it throws an error — so we should really just use 200 here and think of `null` as the successfully resolved edge proposal :p

Reply:
Sounds good.